docstring_parser/parser/__init__.py

```diff
@@ -5,14 +5,23 @@
 from . import google, rest
 from .common import Docstring, ParseError
 
+try:
+    from . import numpy
```

Comment:
Try importing `numpydoc`, not `numpy`. It's `numpydoc` we care about.

Reply:
`numpy` is the name of the submodule (under `docstring_parser.parser`) that contains the parsing for NumPy-style docstrings. Did you mean to say that we should name that submodule `numpydoc`?